wpilibc/src/main/native/cpp/drive/DifferentialDrive.cpp

```diff
@@ -16,11 +16,21 @@
 
 using namespace frc;
 
+WPI_IGNORE_DEPRECATED
```

Comment:
Is this still necessary?

Reply:
It will be if we eventually deprecate it again, and finding where those go was a lot of work.